### PR TITLE
6 view comments

### PR DESCRIPTION
**Pull request by richnw**

### CODE REVIEW

File: src/Components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  if (isLoading) return <h1>Loading...</h1>;
+  return (
+    <main>
+      <ul>
+        {comments.map((comment) => (
+          <li>
+            <p> user: {`${comment.author}`} </p>
+            <p> {`${comment.body}`} </p>
+            <br />
+          </li>
```
Comment (popatre) [inline src/Components/Comments.jsx]: Could you add some feedback to account for articles with no comments?
Something like 'no comments to show'

**Review by popatre — CHANGES_REQUESTED** [review]
- handle no comments
- The design isnt mobile first. You'd be better stacking the image above, instead of to the side, it's too squashed.
- reviews/14 doesnt seem to load - dont know if thats an api issue

File: src/Components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  useEffect(() => {
+    api.fetchComments(reviewID).then(({ comments }) => {
+      setComments(comments);
+      setIsLoading(false);
+    });
+  }, [reviewID]);
+
+  if (isLoading) return <h1>Loading...</h1>;
+  else if (comments.length === 0)
+    return <p>No comments on this review... yet</p>;
```
Comment (roisinoreilly) [inline src/Components/Comments.jsx:19]: nicely done

File: src/Components/Comments.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  const { reviewID } = useParams();
+
+  useEffect(() => {
+    api.fetchComments(reviewID).then(({ comments }) => {
+      setComments(comments);
+      setIsLoading(false);
+    });
+  }, [reviewID]);
+
+  if (isLoading) return <h1>Loading...</h1>;
```
Comment (roisinoreilly) [inline src/Components/Comments.jsx]: I'm not sure a h1 is the best tag to use here in terms of accessibility

File: src/Components/ReviewCard.jsx [hunk 4]
```diff
@@ -1,26 +1,20 @@
...
+        {review.title}
+        <br />
+        <br />
+        user: {review.owner}
+        <br />
+        votes: {review.votes}
+        <img
+          className="reviewImage"
+          src={`${review.review_img_url}`}
+          alt={`${review.title}`}
```
Comment (roisinoreilly) [inline src/Components/ReviewCard.jsx]: nice

**Review by roisinoreilly — APPROVED** [review]
looking much better from a user perspective. We could still do with the ability to navigate back from the single articles but otherwise looking great

**Pull request merged**